### PR TITLE
Add missing test

### PR DESCRIPTION
**Pull request by smoralesd**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000 **N/A**
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes. **N/A**

#### Description of changes

Add missing test for popup action message creator

#### Notes for reviewers

**N/A**

### CODE REVIEW

File: src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts [hunk 1]
```diff
@@ -222,4 +219,24 @@ describe('PopupActionMessageCreatorTest', () => {
...
+            tabId: 1,
+            type: Messages.LaunchPanel.Set,
+            payload,
+        };
+
+        postMessageMock.setup(post => post(It.isValue(expectedMessage))).verifiable(Times.once());
+
+        testSubject.setLaunchPanelType(type);
+
+        postMessageMock.verifyAll();
```
Comment (Shobhit1) [inline src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts]: Its not a blocker or anything, but we can move the `verifyAll` calls to `afterEach()`. That just keeps us from calling this function in each test!

Comment (smoralesd) [reply src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts]: I think `afterEach` should be use for clean up purposes only... `verifyAll` is part of the last step of AAA testing so I prefer it to be explicitly part of the test itself.

File: src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts [hunk 2]
```diff
@@ -21,16 +21,17 @@ import { LaunchPanelType } from '../../../../../../popup/scripts/components/popu
 import { EventStubFactory } from '../../../../common/event-stub-factory';
 
 describe('PopupActionMessageCreatorTest', () => {
-    let eventStubFactory: EventStubFactory;
+    const eventStubFactory = new EventStubFactory();
+    const event = eventStubFactory.createKeypressEvent() as any;
```
Comment (dbjorge) [inline src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts]: >event [](start = 10, length = 5)

Can this name be made a little more descriptive (maybe `stubKeypressEvent`?), now that it's being initialized further away from the use sites?

Comment (smoralesd) [reply src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts]: done

**Pull request merged**